server/models/stepDetail.js

```diff
@@ -0,0 +1,35 @@
+const mongoose = require("mongoose");
+const Schema = mongoose.Schema;
+
+const StepDetailSchema = new Schema({
+  prospects: [
+    {
+      type: {
+        type: String,
+        required: true,
+        enum: ["Email", "Call"],
+        default: "Email"
+      },
+      prospect: { type: Schema.Types.ObjectId, required: true, index: true },
+      status: {
+        type: String,
+        required: true,
+        enum: [
+          "Draft",
+          "Sent",
+          "Delivered",
+          "Opened",
+          "Clicked",
+          "Replied",
+          "Bounced",
+          "Opted out"
+        ],
+        default: "Draft"
+      },
+      subject: { type: String, required: true },
+      content: { type: String, required: true }
```

Comment:
I don't know if we need subject and content for this since wouldn't it just be what's on the step model anyway? I'd probably remove those fields for now

Reply:
Removed